### DISCUSSION TITLE
启动器要求更新 3.51，更新完后直接无法启动

### DISCUSSION THREAD

**Discussion by g1ove** [BakaXL-Support/BakaXL-QA-docs#137] [使用时的疑问] [answered]
![image](https://github.com/BakaXL-Support/BakaXL-QA-docs/assets/172156975/903ed122-8129-4776-b499-a54d1a6db577)

![image](https://github.com/BakaXL-Support/BakaXL-QA-docs/assets/172156975/197e5319-1b03-4102-a78b-3bcb06c858b6)

**Comment by g1ove** [comment]
有没有大神教一下如何解决

**Reply by g1ove** [reply]
直接下载最新版本也是无法启动

**Reply by XiaoMouz** [reply]
删除目录下的 BakaXL 文件夹和 `%appdata%` 文件夹内的 BakaXL 文件夹后，重新启动 BakaXL 试试

**Reply by XiaoMouz** [reply, marked as answer]
当然或许你也需要安装 .Net framework 4.8.1，请参照 https://help.bakaxl.com/v3/ 检查系统环境

**Reply by g1ove** [reply]
将  `.Net framework 4.5.3` 更新到 `4.8` 后可以启动了 感谢

操作系统是 win7
